config/routes.rb

```diff
@@ -0,0 +1,33 @@
+Rails.application.routes.draw do
+  devise_for :users
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  get '/public_recipes', to:'recipes#public_recipes', as: :public_recipes
+
+
+  resources :recipes, only: [:create, :index, :destroy, :show, :new, :update] do 
+    member do
+      patch 'update_privacy', to: 'recipes#update_privacy', as: 'update_recipe_privacy'
+      get '/shopping_list', to:'recipes#shopping_list', as: :shopping_list
+    end
+  end
+
+  resources :recipe_foods, only: [:new, :create, :destroy]
+
+  resources :inventories, only: [:create, :index, :show, :destroy, :new] do
+    resources :inventory_foods, only: [:new, :create, :destroy]
+  end
+
+  resources :inventory_foods do
+    delete 'remove_food', on: :member
+  end
+
```

Comment:
- You have done well, but I suggest your routes can be better implemented, please fix this, having so many resources in your application makes your code difficult to maintain. Please fix this 🙏. You might find this [article](https://guides.rubyonrails.org/routing.html) helpful 👍 👍 
-  Please ensure you don't expose any unnecessary endpoints you didn't use in your application. Exposing endpoints in your application might leave your application prune to attacks , please fix this 😄